### PR TITLE
[Az.StorageSync] Change default parameter set of Invoke-AzStorageSyncChangeDetection

### PR DESCRIPTION
**Pull request by geolivo**
## Description

Changes the default parameter set of Invoke-AzStorageSyncChangeDetection such that full share change detection is used if no parameters are passed (that is, the user will only be prompted for RG, SSS, SG, and CEP name without having to provide DirectoryPath or Paths flags).

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - [Design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1101#issuecomment-1007183393)
    - [X] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/StorageSync/StorageSync/help/Invoke-AzStorageSyncChangeDetection.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.StorageSync.dll-Help.xml
-Module Name: Az.StorageSync
+Module Name: Microsoft.Azure.PowerShell.Cmdlets.StorageSync
```
Comment (anpint) [inline src/StorageSync/StorageSync/help/Invoke-AzStorageSyncChangeDetection.md:3]: Why did this change? Did it come as a result of running the script that regenerates help content?

**Review by anpint — COMMENTED** [review]
Did you update the Invoke-AzStorageSyncChangeDetection.md file manually? If so, please run the script instead to generate help content: https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md#updating-help-after-making-cmdlet-changes

File: src/StorageSync/StorageSync/help/Invoke-AzStorageSyncChangeDetection.md (hunk 1 above)
Comment (geolivo) [reply src/StorageSync/StorageSync/help/Invoke-AzStorageSyncChangeDetection.md:3]: It must have, I didn't edit this file manually. Let me try generating it again

Comment (geolivo) [reply src/StorageSync/StorageSync/help/Invoke-AzStorageSyncChangeDetection.md]: I think I somehow generated the help with the wrong Accounts module, it should be fixed now

**Review by anpint — APPROVED** [review]

**Comment by wyunchi-ms** [comment]
Please take a look at the failure of CI and supress the exceptions in BreakingCchangeIssues.csv

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**